docs/workforce-management-framework/topics/ref-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,69 @@
+[id='ref-introducing-raincatcher-{chapter}']
+= {WFM-RC-NameShort} Reference Material
+
+This section includes:
+
+. xref:highlighting-raincatcher-areas-of-focus-{chapter}[Highlighting {WFM-RC-NameShort} Areas of Focus]
+. xref:describing-raincatcher-technically-{chapter}[Describing {WFM-RC-NameShort} Technically]
+. xref:identifying-raincatcher-repositories-{chapter}[Identifying {WFM-RC-NameShort} Repositories]
+. xref:differentiating-raincatcher-from-tech-preview-{chapter}[Differentiating {WFM-RC-NameShort} from 'Tech Preview']
+
+[id='highlighting-raincatcher-areas-of-focus-{chapter}']
+== Highlighting {WFM-RC-NameShort} Areas of Focus
+
+.Areas of Focus
+|===
+|Area |Focus
+
+|Framework
+|A generic and developer friendly framework (set of classes) for building mobile and website Workforce Management solutions
+
+|Modules
+|Modules specific to {WFM-RC-NameShort}, for example: signature-module, scheduler
+
+|Integrations
+|Integrations to satisfy {WFM-RC-NameShort} needs, for example: authentication, storage, offline capabilities, server side synchronization
+
+|User Interface
+|Graphical User Interfaces are implemented in AngularJS and Angular
+|===
+
+[id='describing-raincatcher-technically-{chapter}']
+== Describing {WFM-RC-NameShort} Technically
+
+* link:https://nodejs.org/en/[Node.js] used for server side code
+* Hybrid mobile development using link:https://www.typescriptlang.org/docs/home.html[TypeScript](cordova/web/angular)
+* Object Oriented approach (classes and interfaces)
+* Focus on module reuse and flexibility
+* Incorporates open-source technologies, for example: 
+
+[id='identifying-raincatcher-repositories-{chapter}']
+== Identifying {WFM-RC-NameShort} Repositories
+ 
+The {WFM-RC-NameShort} is available in 3 github repositories: MongoDb, Keycloak, Passport.js
```

Comment:
Why is there MongoDb, Keycloak and Passport.js here? I think this sentence is referring to core, angular-js and docs?